tests/end-to-end/backup/pkg/common/common.go

```diff
@@ -125,32 +145,30 @@ func RunTest(t *testing.T, mode TestMode) {
 	switch mode {
 	case TestBeforeBackup:
 		for _, e2eTest := range e2eTests {
-			if !e2eTest.enabled {
-				logrus.Infof("Skipping %v", e2eTest.name)
-				continue
-			}
-			convey.Convey(fmt.Sprintf("Create resources for %v", e2eTest.namespace), t, func() {
-				t.Logf("Creating Namespace: %s", e2eTest.namespace)
+			t.Run(e2eTest.name, func(t *testing.T) {
+				if !e2eTest.enabled {
+					t.Skip("Test disabled")
+				}
+				t.Logf("[CreateResources: %s] Creating Namespace: %s\n", e2eTest.name, e2eTest.namespace)
```

Comment:
The `\n` escapes are not needed in this file. `Logf` does [add a newline character](https://golang.org/src/testing/testing.go?s=17293:17328#L459) for you.

Reply:
done